README.md

```diff
@@ -1,10 +1,5 @@
-# Slic3r Manual
+# SuplierSlicer Manual
 
-The first release of this manual was sponsored by Lulzbot - Aleph Objects, Inc. and written by Gary Hodgson.
+This documentation is built on a fork of the Slic3r documentation maintained at http://github.com/alexrj/slic3r. Work on this fork started 1st November 2020, and it should be considered a work in progress and incomplete. 
```

Comment:
Was this intentional?

Reply:
no, sorry. I think it's because i didn't use a special branch, I'll close this old thing.
Sorry for the inconvenience